### PR TITLE
Fixed pep8 whining in module unpackfs

### PR DESCRIPTION
**Pull request by agaida**
Added myself to copyright

### CODE REVIEW

**Comment by adriaandegroot** [comment]
LGTM .. why change it to WIP?

File: src/modules/unpackfs/main.py [hunk 1]
```diff
@@ -258,14 +285,14 @@ def run():
...
                 "globalstorage does not contain a \"rootMountPoint\" key, "
                 "doing nothing")
 
     if not os.path.exists(root_mount_point):
-        return ("Bad mount point for root partition in globalstorage",
-                "globalstorage[\"rootMountPoint\"] is \"{}\", which does not "
-                "exist, doing nothing".format(root_mount_point))
+        return ("Bad mount point for root partition in globalstorage, "
+                "globalstorage[\"rootMountPoint\"] is \"{}\", which does "
+                "not exist, doing nothing".format(root_mount_point))
```
Comment (agaida) [inline src/modules/unpackfs/main.py]: WIP because this little gem - not sure about.

File: src/modules/unpackfs/main.py [hunk 2]
```diff
@@ -258,14 +285,14 @@ def run():
     root_mount_point = globalstorage.value("rootMountPoint")
 
     if not root_mount_point:
-        return ("No mount point for root partition in globalstorage",
+        return ("No mount point for root partition in globalstorage, "
                 "globalstorage does not contain a \"rootMountPoint\" key, "
```
Comment (adriaandegroot) [inline src/modules/unpackfs/main.py]: This is wrong, you're changing a 2-tuple return (errorstring, explanationstring) to a single string. Leave the comma where it was.

File: src/modules/unpackfs/main.py [hunk 3]
```diff
@@ -258,14 +285,14 @@ def run():
...
                 "doing nothing")
 
     if not os.path.exists(root_mount_point):
-        return ("Bad mount point for root partition in globalstorage",
-                "globalstorage[\"rootMountPoint\"] is \"{}\", which does not "
-                "exist, doing nothing".format(root_mount_point))
+        return ("Bad mount point for root partition in globalstorage, "
+                "globalstorage[\"rootMountPoint\"] is \"{}\", which does "
```
Comment (adriaandegroot) [inline src/modules/unpackfs/main.py]: This is wrong, you're changing a 2-tuple return (errorstring, explanationstring) to a single string. Leave the comma where it was.

**Pull request merged**